### DISCUSSION TITLE
Want to change the texts

### DISCUSSION THREAD

**Discussion by NguyenHoangMinhkkkk** [react-native-image-picker/react-native-image-picker#2313] [General]
There are some texts that i want to change for some product purpose.
is there anyway to do it by patch-package or just change them at root somewhere ?

![IMG_BF1E65BE3416-1](https://github.com/react-native-image-picker/react-native-image-picker/assets/30792140/d0b5dd55-9ed8-4b08-a1cd-1b0890f45ff6)

**Comment by nathanfoster048-lab** [comment]
If the text is coming from the library itself, `patch-package` should work fine for a product-specific change. You can edit the installed package under `node_modules/react-native-image-picker`, test the change, then create a patch with `npx patch-package react-native-image-picker`.

Just keep in mind that the patch will need to be regenerated if the library changes the relevant files in a future version.
